src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const getListMen = people.filter(person => person.sex === 'm', 0);
+
+  const listToCalculate = century
+    ? getListMen.filter(
+      person => Math.ceil(person.died / 100) === century)
+    : getListMen;
+
+  const getMenAges = listToCalculate.map((person) => person.died - person.born);
+
+  return getMenAges.reduce((total, item) => total + item) / getMenAges.length;
```

Comment:
You can do this in one iteration. Only using method reduce.
```suggestion
  const getMenAges = listToCalculate.map((person) => person.died - person.born);

  return getMenAges.reduce((person, item) => total + (person.died - person.born)) / getMenAges.length;
```

Reply:
this code can't work? but in common you are right? it can be only reduce )